### DISCUSSION TITLE
Show ping and delay of other users?

### DISCUSSION THREAD

**Discussion by dtinth** [jamulussoftware/jamulus#1951] [UI Related]
Hello, first, thank you for making Jamulus! I have been using it for 6 months now. One thing I noticed is when we onboard new musicians in our [local jamming community](https://www.facebook.com/ClubhouseMusicJammingRoom) or when I join other servers, or one of the most frequently asked questions is “What is your ping and delay?”

This happens so often that I think it might be a good idea to show the ping/delay number somewhere, e.g. maybe in the tooltip that shows when you hover a musician.

**Comment by dcorson-ticino-com** [comment]
Hi dt,
Great that you are enjoying Jamulus.
The ping and delay of the other users is not shown because your local client does not know the ping and delay of the other clients.
To show them it would need new network messages sending that information to all the others, lots of complexity and network usage that can be better used for sound data.
We always try to keep things simple and reduce the network usage.
